TESTS/mbedmicro-rtos-mbed/threads/main.cpp

```diff
@@ -25,7 +25,11 @@
   #error [NOT_SUPPORTED] test not supported
 #endif
 
+#if defined(TARGET_NUCLEO_F070RB) || defined(TARGET_NUCLEO_F072RB)
```

Comment:
The rest of changes are fine. I am not certain about this. Are we expecting to have here ifdef for varios targets .

There is not 2x768bytes available for these 2 targets?